src/plugins/intel_npu/tests/functional/behavior/npuw/llm_accuracy_tests.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2024-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_engine/comparators/nrmse.hpp"
+#include "test_engine/simple_llm_pipeline.hpp"
+#include "intel_npu/npuw_private_properties.hpp"
+#include "openvino/util/shared_object.hpp"
+#include "openvino/util/file_util.hpp"
+
+#include <filesystem>
+#include <algorithm>
+
+#include <gtest/gtest.h>
+
+using namespace testing;
+using namespace ov::npuw::tests;
+using namespace ov::intel_npu::npuw;
+
+// this tests load plugin by library name: this is not available during static linkage
+#ifndef OPENVINO_STATIC_LIBRARY
+#ifdef WITH_CPU_PLUGIN
+#if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
+const char* cpu_plugin_file_name = "openvino_intel_cpu_plugin";
+
+namespace {
+const std::vector<int64_t> What_is_OpenVINO =
+    {529, 29989, 1792, 29989, 29958, 13, 5618, 338, 4673, 29963, 1177,
+     29949, 29973, 2, 29871, 13, 29966, 29989, 465, 22137, 29989, 29958, 13};
+const std::vector<int64_t> OpenVINO =
+    {6585, 29963, 1177, 29949}; 
+
+using LLMTestParams = std::tuple<std::string, ov::AnyMap, 
+                                 std::vector<int64_t>, std::vector<int64_t>>;
+} // anonymous namespace
+
+class LLMAccuracyTestsNPUW : public ::testing::TestWithParam<LLMTestParams> {
+public:
+    void SetUp() override {
+        // NOTE: TEMPLATE plugin in OpenVINO works for ~20 minute to generate
+        //       first token from prefill model and crashes on launch of
+        //       3rd subrequest in generate model.
+        //       There is no such issue with CPU plugin, so CPU plugin was choosen
+        //       for accuracy checks.
+        // Register CPU plugin in OpenVINO:
+        try {
+            core.register_plugin(std::string(cpu_plugin_file_name) + OV_BUILD_POSTFIX, "CPU");
+        } catch (ov::Exception& ex) {
+            if (std::string{ex.what()}.find("Device with \"CPU\"  is already registered in the OpenVINO Runtime")
+                == std::string::npos) {
+                throw ex;
+            }
+        }
+
+        auto param = GetParam();
+        ov::AnyMap config;
+        std::tie(model_path, config, input_ids, reference_ids) = param;
+        config["NPUW_DEVICES"] = "CPU";
+        config["NPUW_LLM_MIN_RESPONSE_LEN"] = 4;
```

Comment:
what's the prompt length? Not sure why you limit this one here, but if your test prompt is short you may want to limit the PROMPT_LEN as well to improve the prefill's performance on CPU.